packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -322,6 +331,11 @@ export class TaskDefinition extends TaskDefinitionBase {
    */
   private readonly placementConstraints = new Array<CfnTaskDefinition.TaskDefinitionPlacementConstraintProperty>();
 
+  /**
+   * Inference accelerators for task instances
+   */
+  public readonly inferenceAccelerators: InferenceAccelerator[] = [];
```

Comment:
Is there any reason why we want to expose this class member? Reason why i ask is because we should be very careful creating a public class member since it would be a breaking change if we want to make it private again.

Reply:
It was set to public as it was being referenced in the container definition. But making it private with a public getter method would make more sense. Will make the required changes.